.github/workflows/lint_check.yaml

```diff
@@ -16,11 +16,11 @@ jobs:
         python_version: ["3.10"]
     steps:
       - name: Set up python ${{ matrix.python_version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python_version }}
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

Comment:
Lint check seems to be complaining about this line

Reply:
https://github.com/alteryx/featuretools_sql/pull/75/commits/777a60252d7b2c887901f758d14d293ddb875f15